post_installation_configuration/bare-metal-configuration.adoc

```diff
@@ -0,0 +1,25 @@
+:_content-type: ASSEMBLY
+:context: post-install-bare-metal-configuration
+[id="post-install-bare-metal-configuration"]
+= Bare metal configuration
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+When deploying {product-title} on bare metal hosts, there are times when you need to make changes to bare metal hosts either before or after provisioning. This can include inspecting the host's hardware, firmware, and BIOS details. It can also include cleaning disks or changing modifiable BIOS settings. There are two resources that you can use with the Bare Metal Operator (BMO):
+
+* `BareMetalHost`
+* `HostFirmwareSettings`
+
+There is also a `FirmwareSchema` resource, which you must use to validate that you are sending correct values to a host when making changes to host firmware settings.
```

Comment:
Recommend changing this to "can use".  The user can use it when setting the new values, while the BMO will use it to check validity.